drivers/rtc/rtc_pcf8563.c

```diff
@@ -219,6 +230,11 @@ static int pcf8563_alarm_set_time(const struct device *dev, uint16_t id, uint16_
 		return -EINVAL;
 	}
 
+	if (!rtc_utils_validate_rtc_time(timeptr, PCF8563_RTC_ALARM_TIME_MASK)) {
```

Comment:
Only the fields set by the `mask` arg need to be validated :)
```
if (!rtc_utils_validate_rtc_time(timeptr, mask)) {
```

Reply:
oops, my bad